app/src/controllers/testItem/sagas.js

```diff
@@ -121,7 +122,7 @@ function* fetchTestItems({ payload = {} }) {
   );
   const dataPayload = yield take(createTestItemActionPredicate(NAMESPACE));
   const currentLevel = yield select(levelSelector);
-  const level = calculateLevel(dataPayload.payload.content) || currentLevel;
+  const level = calculateLevel(dataPayload.payload.content) || currentLevel || LEVEL_SUITE;
```

Comment:
1. Please rebase, this line was updated with other navigation fix.
2. This fix won't work for the case when we open a link with wrong item id. There should be special error screen for that (#1238).